server/server.go

```diff
@@ -958,7 +957,7 @@ func (s *Server) Start() error {
 
 	ctx, _ := context.WithTimeout(context.Background(), 5*time.Second) // nolint: vet
 	if err := server.Shutdown(ctx); err != nil {
-		return cli.NewExitError(fmt.Sprintf("while shutting down: %s", err), 1)
+		return fmt.Errorf("while shutting down: %s", err)
```

Comment:
atlantis use cobra, so `cli.ExitCoder` is handled as simple err.

https://github.com/urfave/cli/blob/8c0fb2dba4f52d1a93101ee529a490da1dcb6718/errors.go#L77-L96